tests/integration-tests/tests/api/test_api_client.py

```diff
@@ -47,6 +65,12 @@ def test_cluster_operations(request, scheduler, region, pcluster_config_reader,
 
     _test_list_clusters(client, cluster, region)
     _test_describe_cluster(client, cluster, region)
```

Comment:
can we run a describe cluster also right after the create and assert the status?